test/e2e/serverside_apply_test.go

```diff
@@ -65,7 +65,7 @@ func serversideApplyTest(c client.Client, invConfig InventoryConfig, inventoryNa
 	}, apiService)
 	Expect(err).NotTo(HaveOccurred())
 	_, found2 := apiService.GetAnnotations()[v1.LastAppliedConfigAnnotation]
-	Expect(found2).To(Equal(true))
+	Expect(found2).To(BeFalse())
```

Comment:
Adding a comment that `LastAppliedConfigAnnotation` is only set for client-side apply, and we've server-side applied here would make this easier for others to follow.